app/models/user.js

```diff
@@ -23,10 +23,17 @@ module.exports = (sequelize, DataTypes) => {
     return bcrypt.compareSync(password, this.password);
   };
 
+  User.prototype.toJSON = function() {
```

Comment:
dont overwrite object prototypes

Reply:
which is the best way to hide the encrypted password that is sent when the user is returned?